crypt/cipher.go

```diff
@@ -284,11 +293,182 @@ func (c *cipher) decryptSegment(ciphertext string) (string, error) {
 	return string(plaintext), err
 }
 
+// Simple obfuscation routines
+
+func (c *cipher) obfuscateSegment(plaintext string) string {
+	if plaintext == "" {
+		return ""
+	}
+
+	// If the string isn't valid UTF8 then don't rotate; just
+	// prepend a 0.
+	if !utf8.ValidString(plaintext) {
+		return "0." + plaintext
+	}
+
+	// Calculate a simple rotation based on the filename and
+	// the nameKey
+	var dir int
+	for _, runeValue := range plaintext {
+		dir += int(runeValue)
+	}
+	dir = dir % 256
+
+	// We'll use this number to store in the result filename...
+	result := strconv.Itoa(dir) + "."
+
+	// but we'll augment it with the nameKey for real calculation
+	for i := 0; i < len(c.nameKey); i++ {
+		dir += int(c.nameKey[i])
+	}
+	dir = dir % 127
```

Comment:
I was doing `% 256` earlier just to keep the numbers small in the filename but without impacting the total distance calculation.  The `% 127` was to make it suitable for the 256-block range (1/2 range size -1).  It should probably be moved into that case section to match the code logic used for the ASCII range.  That may change the test results; I'll update the unit tests if that happens.